cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -65,6 +72,11 @@ type node struct {
 	tenantCRSs map[*node]empty
 }
 
+type crdInfo struct {
```

Comment:
What about using 
```suggestion
type typeInfo struct {
```
or
```suggestion
type discoveryTypeInfo struct {
```
considered that in the list of types we are already including Secret and ConfigMap, which are not CRDs